pymongoexplain/__main__.py

```diff
@@ -13,48 +13,35 @@
 # limitations under the License.
 
 
-from pymongo import monitoring, MongoClient
 from pymongo.collection import Collection
+from .explainable_collection import ExplainCollection
+
 import sys
-from bson.son import SON
+import logging
 
-class CommandLogger(monitoring.CommandListener):
-    def __init__(self):
-        self.payloads = []
+FORMAT = '%(asctime)s %(levelname)s %(module)s %(message)s'
+logging.basicConfig(format=FORMAT, level=logging.INFO)
```

Comment:
We need to investigate how us configuring logging in `__main__` might interact with the script being run configuring logging as well. I think if it is technically feasible, we should try to ensure:

* that the logging configuration in __main__  does not make the logger less verbose than as configured by the script, and
* that the script cannot configure logging in a manner that would inadvertently suppress explain output.

I don't necessarily think that these objectives need to be achieved in code - good documentation should also suffice. Please open an issue to track this work.